supply-chain/audits.toml

```diff
@@ -70,6 +70,12 @@ who = "David Cook <dcook@divviup.org>"
 criteria = "safe-to-run"
 delta = "0.21.4 -> 0.21.5"
 
+[[audits.base64]]
+who = "Brandon Pitman <bran@bran.land>"
+criteria = "safe-to-run"
+delta = "0.21.5 -> 0.21.6"
+notes = "sourcegraph-based diff did not see the v0.21.6 tag; I retrieved a local copy of the repo and used that for diff'ing."
```

Comment:
FYI, the sourcegraph URLs are from crates.io archives, rather than from VCS. It looks like it's caught up now: https://sourcegraph.com/crates/base64/-/compare/v0.21.5...v0.21.6?visible=9. If you run into issues in the future, there's also `cargo vet diff --mode=local`, which will unzip them locally and execute `git diff --no-index`.